HeartBeat.php

```diff
@@ -1,25 +1,36 @@
 <?php
-
+/**
+ * HeartBeat Plugin for LimeSurvey
+ * A heartbeat plugin for LimeSurvey to ensure the session is kept alive.
+ *
+ * @author Frederik Prijck <http://www.frederikprijck.net/>
+ * @author Denis Chenu <denis@sondages.pro>
+ * @copyright 2016 Frederik Prijck <http://www.frederikprijck.net/>
+ * @license MIT
+ * @license https://opensource.org/licenses/MIT MIT License (MIT)
+ * @version 1.0.1
+ *
+ */
 class HeartBeat extends \ls\pluginmanager\PluginBase {
 
     static protected $name = 'HeartBeat';
     static protected $description = 'Implement a heartbeat to ensure the session is kept alive.';
 
     protected $storage = 'DbStorage';
     protected $settings = array(
+        'useSessionLifeTime'=>array(
```

Comment:
I move it at top : seems better , no ?
Maybe add an help : it use the actual session life time each time it was called , or something like this. Something to explain why it's better :)